data/json/recipes/recipe_others.json

```diff
@@ -4345,18 +4345,7 @@
     "reversible": true,
     "time": "1 s",
     "autolearn": true,
-    "components": [ [ [ "washboard", 1 ] ], [ [ "sponge", 1 ], [ "washcloth", 1 ] ] ]
-  },
-  {
-    "type": "recipe",
-    "result": "washcloth",
```

Comment:
You need to move recipe to `/recipe/recipe_obsolete.json` with `obsolete: true` and item to `/items/obsolete.json`.